docs/src/assets/readme.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
```

Comment:
do not create the second file with the same contents as README.md in this directory

Reply:
The comment still applies.

Reply:
Just add the comment about the Risk column to the already existing file.